lib/jsonapi/view.ex

```diff
@@ -88,7 +88,8 @@ defmodule JSONAPI.View do
   """
   defmacro __using__(opts \\ []) do
     {type, opts} = Keyword.pop(opts, :type)
-    {namespace, _opts} = Keyword.pop(opts, :namespace, "")
+    {module_namespace, _opts} = Keyword.pop(opts, :namespace)
+    namespace = module_namespace || Application.get_env(:jsonapi, :namespace, "")
```

Comment:
A humble request: would you mind documenting this change in the README (or elsewhere) too?